src/game/features/players/toxic/Ragdoll.cpp

```diff
@@ -0,0 +1,21 @@
+#include "core/frontend/Notifications.hpp"
+#include "game/commands/PlayerCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Natives.hpp"
+
+
+namespace YimMenu::Features
+{
+
+	class Ragdoll : public PlayerCommand
+	{
+		using PlayerCommand::PlayerCommand;
+
+		virtual void OnCall(Player player) override
+		{
+			PED::SET_PED_TO_RAGDOLL(PLAYER::GET_PLAYER_PED(player.GetId()), 5000, 5000, 0, true, true, nullptr);
```

Comment:
Does this work on player peds?

Reply:
I bet there is a net event that does work, just like on gta.

Reply:
There is a net event, but do we have a way to send the event currently?

Reply:
> There is a net event, but do we have a way to send the event currently?

No, but since this won't do anything having it now has no use.

Reply:
alright i will remove the rag doll feature